pkgs/data/themes/matcha/default.nix

```diff
@@ -1,14 +1,14 @@
 { stdenv, fetchFromGitHub, gdk-pixbuf, librsvg, gtk-engine-murrine }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "matcha";
```

Comment:
```suggestion
  pname = "matcha-unstable";
```

Reply:
This is a released version. It is not just a revision of the repository.